lizmap/modules/lizmap/classes/qgisFormControl.class.php

```diff
@@ -258,7 +261,11 @@ public function __construct($ref, $edittype, $prop, $aliasXml = null, $defaultVa
                     $markup = $this->qgisEdittypeMap[$this->fieldEditType]['jform']['markup'][0];
                 }
             } elseif ($this->fieldEditType === 'TextEdit') {
-                if (property_exists($this->widgetv2configAttr, 'UseHtml')) {
+                if (property_exists($this->widgetv2configAttr, 'IsMultiline')
+                    && ($this->widgetv2configAttr->IsMultiline === '0' || $this->widgetv2configAttr->IsMultiline === 'false')) {
+                    $this->fieldEditType = 'LineEdit';
+                    $markup = $this->qgisEdittypeMap[$this->fieldEditType]['jform']['markup'];
+                } else if (property_exists($this->widgetv2configAttr, 'UseHtml')) {
                     $markup = $this->qgisEdittypeMap[$this->fieldEditType]['jform']['markup'][(int) $this->widgetv2configAttr->UseHtml];
                 } else {
                     $markup = $this->qgisEdittypeMap[$this->fieldEditType]['jform']['markup'][0];
```

Comment:
@rldhont line 276 : allowMulti is now a boolean. Changing cast (int) to (bool) works.

```xml
        <field name="name">
          <editWidget type="ValueRelation">
            <config>
              <Option type="Map">
                <Option type="bool" name="AllowMulti" value="true"/>
                <Option type="bool" name="AllowNull" value="true"/>
                <Option type="QString" name="FilterExpression" value=""/>
                <Option type="QString" name="Key" value="id"/>
                <Option type="QString" name="Layer" value="test_layer_7714a8b5_23f3_4fe1_a100_af6b23df6940"/>
                <Option type="int" name="NofColumns" value="1"/>
                <Option type="bool" name="OrderByValue" value="false"/>
                <Option type="bool" name="UseCompleter" value="false"/>
                <Option type="QString" name="Value" value="name"/>
              </Option>
            </config>
          </editWidget>
        </field>
```

Reply:
AllowMulti is not yet used

Reply:
@rldhont Allowmulti was supported.
Here is my fix, it works with '0', '1', 'false' and 'true' : 
`$markup = $this->qgisEdittypeMap[$this->fieldEditType]['jform']['markup'][(int) filter_var($this->widgetv2configAttr->AllowMulti, FILTER_VALIDATE_BOOLEAN) ];`

Reply:
Done